src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenOverridingAndHiding.cs

```diff
@@ -4121,13 +4122,31 @@ static void Main()
 
                 if (isFromMetadata)
                 {
-                    WellKnownAttributesTestBase.VerifyParamArrayAttribute(parameterB);
+                    VerifyParamArrayAttribute(parameterB);
                 };
             };
 
             var verifier = CompileAndVerify(source, symbolValidator: validator(true), sourceSymbolValidator: validator(false), expectedOutput: @"System.Int32[]");
         }
 
+        private static void VerifyParamArrayAttribute(ParameterSymbol parameter, bool expected = true)
```

Comment:
Original declaration for reference

https://github.com/dotnet/roslyn/blob/f51c1b219058583b56aa826d8ed89a9d92495243/src/Compilers/CSharp/Test/Emit/Attributes/WellKnownAttributesTestBase.cs#L86-L103